delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/OrganizationsRoutesSpec.scala

```diff
@@ -77,7 +81,23 @@ class OrganizationsRoutesSpec
 
   "An OrganizationsRoute" should {
 
+    "fail to create an organization without organizations/write permission" in {
```

Comment:
You can use scalatest inspectors to test in a single test all the paths where write permissions is needed, another single test for all tests where read is needed as the output is always the same